### PR TITLE
SUL23-525: Updates to the display for Med breakpoint on Branches and Centers

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Moved the medium breakpoint to a grid display to get the correct design. Originally we talked about using flex instead but it wasn't working to get the right display. The other option was `floats` and I think those are even less liked.
- I would like to use this technique for the Places to Study and People on the medium breakpoints. Both of those tables have the same spacing issue. I thought it was best to have it code reviewed in one place and then I can move the change to the other two tables.

# Review By (Date)
- 7/18

# Criticality
- High

# Urgency
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci` 
3. Navigate to a list of Branches on a medium breakpoint.
4. Verify it looks like the design.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory?

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- SUL23-525](https://stanfordits.atlassian.net/browse/SUL23-525)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: src/components/views/sul-branch-locations/filtering-table/branch-locations-filtering-table.tsx [hunk 1]
```diff
@@ -121,9 +121,9 @@ const TableRow = ({id, imageUrl, path, title, phone, email, mapUrl, address, hou
   return (
     <Tr
       key={id}
-      className="block sm:flex-col sm:flex-wrap sm:text-center md:flex md:max-h-[400px] md:text-left lg:!table-row lg:max-h-none"
+      className="block sm:flex-col sm:flex-wrap sm:text-center md:grid md:grid-cols-2 md:grid-rows-[repeat(4,minmax(0,auto))] md:justify-items-start md:gap-x-20 md:text-left md:align-top lg:!table-row lg:max-h-none"
```
Comment (pookmish) [inline src/components/views/sul-branch-locations/filtering-table/branch-locations-filtering-table.tsx:124]: I wasn't sure about the whole `grid-rows` until I started to look at it.

**Review by pookmish — APPROVED** [review]
LGTM. Try to avoid floats and inlines as much as we can. Grid is much bettter.

**Comment by jenbreese** [comment]
@pookmish Oh great. I'm glad you looked at this.  I updated the two other tablular data pages to use the grid at the md breakpoint too. @imonroe can you take a look at your list page for those other two list and let me know if they look ok too?

**Comment by imonroe** [comment]
Looking at them here: https://su-library-d983xys6o-stanford-libraries.vercel.app/test-list-page
looks consistent

**Pull request merged**